python/GafferArnoldUI/ArnoldAttributesUI.py

```diff
@@ -239,6 +251,27 @@ def __volumeSummary( plug ) :
 
 		],
 
+		# Transform
+	
+		"attributes.transformType" : [
+
+			"description",
+			"""
+			Choose how transform motion is interpolated.  Linear produces classic Renderman-style linear vertex motion, rotate_about_origin produces curves arcs centerd on the object's origin, and rotate_about_center, the default, produces curved arcs centred on the object's bounding box middle.
```

Comment:
- Let's drop the reference to RenderMan - there was nothing in the standard to require linear motion, some renderers did curved motion, and the standard isn't even standard any more.
- I presume you mean "curved arcs" rather than "curves arcs", although maybe one of those words is redundant anyway?
- "centerd" -> "centred"
- But of line wrapping?